### ISSUE TITLE
Odroid: diskLayout might use the wrong case in filter

### ISSUE THREAD

**Issue by MauriceNino** [sebhildebrandt/systeminformation#695]
**Describe the bug**
A user of my software does not get `diskLayout` to recognize his disk (https://github.com/MauriceNino/dashdot/issues/111).

I think this is the relevant part of the output (but you can look at the linked issue to get the full output):

```
{
  "alignment": 0,
  "disc-aln": 0,
  "dax": false,
  "disc-gran": 4194304,
  "disc-max": 3741319168,
  "disc-zero": false,
  "fsavail": null,
  "fsroots": [null],
  "fssize": null,
  "fstype": null,
  "fsused": null,
  "fsuse%": null,
  "fsver": null,
  "group": "disk",
  "hctl": null,
  "hotplug": true,
  "kname": "mmcblk0",
  "label": null,
  "log-sec": 512,
  "maj:min": "179:0",
  "min-io": 512,
  "mode": "brw-rw----",
  "model": null,
  "name": "mmcblk0",
  "opt-io": 0,
  "owner": "root",
  "partflags": null,
  "partlabel": null,
  "parttype": null,
  "parttypename": null,
  "partuuid": null,
  "path": "/dev/mmcblk0",
  "phy-sec": 512,
  "pkname": null,
  "pttype": "dos",
  "ptuuid": "cbf654cd",
  "ra": 128,
  "rand": false,
  "rev": null,
  "rm": false,
  "ro": false,
  "rota": false,
  "rq-size": 64,
  "sched": "none",
  "serial": "0xaceb3869",
  "size": 127865454592,
  "start": null,
  "state": null,
  "subsystems": "block:mmc:mmc_host:platform",
  "mountpoint": null,
  "mountpoints": [null],
  "tran": null,
  "type": "disk",
  "uuid": null,
  "vendor": null,
  "wsame": 0,
  "wwn": null,
  "zoned": "none",
  "zone-sz": 0,
  "zone-wgran": null,
  "zone-app": 0,
  "zone-nr": 0,
  "zone-omax": 0,
  "zone-amax": 0
}
```

The problem is, that you check for `fsType`, but it seems to be `fstype` in the user's output: https://github.com/sebhildebrandt/systeminformation/blob/b8b37f6f899d8e5cd02238f9a030172259f341d4/lib/filesystem.js#L855

All other things in this output seem to be valid.

**Environment (please complete the following information):**
 - systeminformation package version: 5.11.16
 - OS: Odroid
 - Hardware Odroid N2+

**Comment by sebhildebrandt** [comment]
@MauriceNino you are absolutely right. Provided a fix for the filtering. Hope this fixes the bug also on your side. Version 5.11.17 just released.

**Comment by MauriceNino** [comment]
@sebhildebrandt I think this needs to be re-opened.

Seems like now some false positives go through as well. Another user of my software has created an issue (https://github.com/MauriceNino/dashdot/issues/114) which now detects a lot of drives, but most of them are not really hard drives as it seems.

Maybe the filter needs more fixing?

```
[
  {
    device: '/dev/ram0',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram1',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram2',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram3',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram4',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram5',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram6',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram7',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram8',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram9',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram10',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram11',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram12',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram13',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram14',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/ram15',
    type: 'SSD',
    name: '',
    vendor: '',
    size: 4194304,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/sda',
    type: 'SSD',
    name: 'SSD 870 EVO 500G',
    vendor: 'Samsung',
    size: 500107862016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '0',
    serialNum: '',
    interfaceType: 'USB',
    smartStatus: 'unknown',
    temperature: null
  }
]
```

**Comment by sebhildebrandt** [comment]
@MauriceNino we would absolutely need the output of `lsblk -ablJO` from this specific machine ... is it possible to provide it?

**Comment by MauriceNino** [comment]
I asked the user that created the issue and will notify you when I have any updates.

**Comment by sebhildebrandt** [comment]
@MauriceNino ... I created a small fix that limits the false positives .... but still we would need the output. Thank you for your help on this one. Version 5.11.18 just released.

**Comment by sebhildebrandt** [comment]
@MauriceNino based on the provided output on your side I now made a fix that should work. Version 5.11.19 just released.

**Comment by MauriceNino** [comment]
Nice, but are you sure it is correct? The `fsType` seems to be in the wrong case again, or is this intentional?

**Comment by sebhildebrandt** [comment]
@MauriceNino oh man ... my fault again ... fixed, version 5.11.20 just released.

**Comment by MauriceNino** [comment]
Nice, thank you again!
